daemon/graphdriver/driver_windows.go

```diff
@@ -0,0 +1,26 @@
+package graphdriver
+
+type DiffDiskDriver interface {
+	Driver
+	CopyDiff(id, sourceId string) error
+}
+
+const (
+	FsMagicWindows = FsMagic(0xa1b1830f)
+)
+
+var (
+	// Slice of drivers that should be used in an order
+	priority = []string{
+		"windows",
+	}
+
+	FsNames = map[FsMagic]string{
+		FsMagicWindows:     "windows",
```

Comment:
This will be updated later once we are ready to submit a PR with the full graph driver implementation.  For now, this is just a placeholder.